web/components/contract/contract-details.tsx

```diff
@@ -162,12 +143,12 @@ export function ContractDetails(props: {
         isMobile ? 'max-w-[140px]' : 'max-w-[250px]'
       )}
     >
-      <SiteLink href={groupPath(groupToDisplay.slug)} className={'truncate'}>
-        <Row>
+      <Link prefetch={false} href={groupPath(groupToDisplay.slug)}>
```

Comment:
I don't really think the `SiteLink` -> `MaybeLink` abstraction makes any sense to use when we know in advance whether or not the link is to something on Manifold.

Reply:
I think I created `SiteLink` and `ExternalLink` once. `SiteLink` made sure to use `Link` and `a`, whereas `ExternalLink` just used `a` and opened in a new tab.

If `SiteLink` were still internal we could add a prefetch prop, but oh well I guess this is fine.